.github/workflows/CICD.yml

```diff
@@ -395,6 +399,10 @@ jobs:
         persist-credentials: false
     - uses: dtolnay/rust-toolchain@stable
     - uses: Swatinem/rust-cache@v2
+    - name: Install/setup prerequisites
+      shell: bash
+      run: |
+        sudo apt-get -y update ; sudo apt-get -y install libselinux1-dev
```

Comment:
Is it on purpose that this is its own step and not merged with the "Install dependencies" step (lines 408 - 413)?

Reply:
missed it :)